src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -453,7 +453,10 @@
     <value>Invalid data at offset {0}: {1}{2}*{3}{4}</value>
   </data>
   <data name="SymWriterNotDeterministic" xml:space="preserve">
-    <value>SymWriter doesn't support deterministic compilation</value>
+    <value>Windows PDB writer doesn't support deterministic compilation (is the compiler host missing Microsoft.DiaSymReader.Native library?)</value>
+  </data>
+  <data name="SymWriterNotAvailable" xml:space="preserve">
+    <value>Windows PDB writer is not available (is the compiler host missing Microsoft.DiaSymReader.Native library?)</value>
```

Comment:
I'm concerned that "compiler host" isn't a common enough term, but I don't have a better suggestion. We can use it for RC2 and think of something better later, as Andy suggests.